octis/optimization/optimizer_evaluation.py

```diff
@@ -21,8 +22,11 @@ def __init__(self, optimizer, BO_results):
         dict_metric_parameters = dict()
 
         for mp in metric_parameters:
-            if check_instance(getattr(optimizer.metric, mp)):
-                dict_metric_parameters.update({mp: getattr(optimizer.metric, mp)})
+            for mp in (metric_parameters):
```

Comment:
Why is the second loop over `metric_parameters` needed?

Reply:
sorry, it is not needed.